test/test_jit.py

```diff
@@ -2021,6 +2021,28 @@ def checkScript(self, script, inputs, optimize=True, outputs=None, name='func',
 
         return ge
 
+    def test_tuple_io(self):
+        def stuff(x):
+            # type: (Tuple[Tensor, Tensor]) -> Tuple[Tensor, Tensor]
+            a, b = x
+            return b, a
+
+        a = (torch.ones(3), torch.ones(3))
```

Comment:
It would be better to use distinct values for this test, because a function that returns `x` unchanged would pass it too.